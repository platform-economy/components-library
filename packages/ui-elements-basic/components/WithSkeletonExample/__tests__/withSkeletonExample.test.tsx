import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
/*
A set of utilities for testing Styled Components with Jest.
This package improves the snapshot testing experience and
provides a brand new matcher to make expectations on the style rules.
*/
import 'jest-styled-components';

// Component
import { WithSkeletonExample } from '../index';

describe('ui-elements-basic', () => {
  describe('Components', () => {
    describe('WithSkeletonExample', () => {

      it('should render with default properties only', () => {
        const tree = TestRenderer
          .create(<WithSkeletonExample />)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
