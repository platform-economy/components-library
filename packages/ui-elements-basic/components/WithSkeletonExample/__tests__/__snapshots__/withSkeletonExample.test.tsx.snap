// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-elements-basic Components WithSkeletonExample should render with default properties only 1`] = `
<div>
  WithSkeletonExample
</div>
`;
